tests/integrated/test-laplacexy/runtest

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
+
+# 
+# Run the test, compare results against the benchmark
+#
+
+# Variables to compare
+tol = 4e-12                  # Absolute tolerance
+
+from boututils.run_wrapper import shell, shell_safe, launch_safe, getmpirun
+from boutdata.collect import collect
+import numpy as np
+from sys import stdout, exit
+
+MPIRUN = getmpirun()
+
+vars = ["check", "check2"]
+
+print("Making LaplaceXY inversion test")
+shell("rm test-laplacexy")
```

Comment:
Probably better to drop this, as the executable might have just been built with `make build-check-tests` or similar